src/pineko/check.py

```diff
@@ -65,3 +65,31 @@ def check_grid_and_eko_compatible(pineappl_grid, operators, xif):
     # x-grid
     if not np.all(in1d(np.unique(operators["targetgrid"]), np.array(x_grid))):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
+
+
+def check_grid_contains_fact_sv(grid):
+    """Checks whether factorization scale-variations are available in the pineappl grid.
+    Parameters
+    ----------
+        grid: pineappl.grid.Grid
+            Pineappl grid
+    """
+    order_list = [order.as_tuple() for order in grid.orders()]
+    for order in order_list:
+        if order[-1] != 0:
+            return
+    raise ValueError("Factorization scale variations are not available for this grid")
+
+
+def check_grid_contains_ren_sv(grid):
```

Comment:
This and the former function I'd slightly improve the name:
- `check` is already the name of the file, and since it is already the namespace there is no need to repeat
- `grid` is already the object is acting on (so little to no need even here, I'd drop)
- `sv` is already implicit in `ren` and `fact`, no need to repeat

In the end they will be `contains_fact` and `contains_ren`. I'd like to avoid even the contain, with a more pythonic construction like
```py
"factoriziation" in grid
```
(`in` is actually the operator `__contains__`, so it's perfect), but I see plenty of problems. First you have to overwrite the default behavior of the `pinappl.grid.Grid.__contains__`, that is already defined somewhere else.

So fine by `contains_fact` and `contains_ren`.